src/web/mjs/connectors/ExHentai.mjs

```diff
@@ -1,13 +0,0 @@
-import EHentai from './EHentai.mjs';
-
-export default class ExHentai extends EHentai {
-
-    constructor() {
-        super();
-        super.id = 'exhentai';
-        super.label = 'ExHentai';
-        this.tags = ['hentai', 'multi-lingual'];
-        this.url = 'http://exhentai.org';
```

Comment:
This website is part of the hentai@home network and requires an authorized account to be accessed.
Should not be deleted for now (even so there is currently an IP banning problem for fast downloads...).

Reply:
I'm unable to access 'exhentai' from anywhere. Are you sure this connector is valid? How do you connect?

Reply:
Login with account on their [forum](https://forums.e-hentai.org/) to get a cookie. With this cookie set in your browser the website became available...

![image](https://user-images.githubusercontent.com/27136773/94162116-a83f1280-fe86-11ea-8cba-9b64395ce623.png)

Reply:
Fair enough.